spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -287,7 +287,8 @@ public Scan buildCopyOnWriteScan() {
             .ignoreResiduals()
             .caseSensitive(caseSensitive)
             .filter(filterExpression())
-            .project(expectedSchema);
+            .project(expectedSchema)
+            .includeColumnStats();
```

Comment:
Why do we need this?

Reply:
Not needed anymore, thanks. I was initially using stats to check if files are already sorted as I didn't have sort ids in data files.